mlrun/track/base_tracker.py

```diff
@@ -12,24 +12,57 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import abc
+import importlib
+import pathlib
+from abc import abstractmethod
 
 from mlrun.config import config as mlconf
-from .tracker import Tracker
 
-from mlrun.execution import MLClientCtx
+from .tracker import Tracker
 
 
 class BaseTracker(Tracker):
-    MODULE_NAME = ...
+    TRACKED_MODULE_NAME = ...
+
+    def __init__(self):
+        super().__init__()
+        self._tracked_platform = importlib.import_module(self.TRACKED_MODULE_NAME)
+        self._run_track_kwargs = {}
+        self._artifacts = {}
+
+    @classmethod
+    def is_enabled(cls) -> bool:
+        try:
+            # Check if the module is available - can be imported:
+            importlib.import_module(cls.TRACKED_MODULE_NAME)
+
+            module_in_area = True
+        except Exception:
+            module_in_area = False
 
-    def is_enabled(self) -> bool:
+        # Check for user configuration - defaulted to False if not configured:
+        return module_in_area and (
+            getattr(mlconf.tracking, cls.TRACKED_MODULE_NAME, {}).mode == "enabled"
+        )
```

Comment:
I would first check in the `mlconf` before trying to import something. And, please see how `mlconf` is used, I think it is creating the fields automatically, so you don't need to `getattr` anything.

Reply:
`cls.TRACKED_MODULE_NAME` is just a `str`, you can access it whenever you want :)